data/tooltips.txt

```diff
@@ -496,3 +496,6 @@ tip "no fuel?"
 
 tip "insufficient energy to fire?"
 	`This ship has insufficient energy storage to fire an installed weapon.`
+
+tip "negative attribute!"
```

Comment:
Unrelated change. But as stated, this isn't even necessarily something that needs changed. Ref: https://github.com/endless-sky/endless-sky/issues/1402#issuecomment-239461582

Reply:
> Unrelated change

It is related in that the sanity check would have prevented Blop from launching with more ammo than he has capacity for. I agree that it's a separate change and that's why I put it in a separate commit (which I have now removed). Thanks for the reference.